codeforboston/maple
# Update view profile on close

# Summary

_Add a short summary of the changes, and a reference to the original issue using `#` and the issue number, like #1_

# Checklist

- N/A On the frontend, I've made my strings translate-able.
- N/A If I've added shared components, I've added a storybook story.
- N/A I've made pages responsive and look good on mobile.

# Screenshots

Viewing a profile that does not belong to the account that is signed in:

![image](https://github.com/codeforboston/maple/assets/73559781/f0597858-d7cf-47a5-b985-a3cf5e423f3e)

After using View Profile navlink, signed in account profile is properly rendered:

![image](https://github.com/codeforboston/maple/assets/73559781/f04446d4-0d6e-4d81-b5be-895721ba65b9)

# Known issues

None

# Steps to test/reproduce

1. Sign into an account
2. navigate to the profile page of an account other than the one that is signed in (possible test account on vercel deployment -> https://maple-emu0mv7x5-maple-testimony.vercel.app/profile?id=8P5Ar8NyAcNEVJMTexezXvRbept2)
3. click on right side of the Navbar where the signed in account name is shown
4. click on View Profile
5. verify if page reload and displays the profile page of the signed in account / if nothing happens and the profile page of the non-signed in account still displays and the proper page doesn't load, the code has failed
